vis/vis.cc

```diff
@@ -1137,7 +1137,7 @@ void LoadPortals(char *name, mbsp_t *bsp)
 
     /* Load the cluster expansion map if needed */
     if (bsp->loadversion->game->id == GAME_QUAKE_II) {
-        clustermap = new int[portalleafs_real];
```

Comment:
Careful with this change, `bsp->numleafs` includes all of the submodels, as well as the reserved leaf 0.
`portalleafs_real` (iirc) doesn't include either of those, so it's always at least 1 smaller than `bsp->numleafs`.

I guess it won't strictly break anything but accessing outside of 0..`portalleafs_real-1` should probably trigger a crash.

Reply:
Yeah it did crash - the issue I was trying to solve here was that the cluster expansion map does not seem to work properly with a PRT1/PRT2 from ericbsp. I don't know exactly why yet.